access.lua

```diff
@@ -21,6 +21,33 @@ local rex = require("rex_pcre")
 local config = require("config")
 local conf = config.get_config()
 
+-- Cache expensive calculations
+local cache = ngx.shared.cache
+
+-- Hash a string using hmac_sha512, return a hexa string
+function cached_jwt_verify(data, method, secret)
```

Comment:
```suggestion
function cached_jwt_verify(data, secret)
```

`method` doesn't seem to be used, so let's just assume we're using HS256 everywhere